app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -414,7 +417,15 @@ function AcuantCapture(
    */
   function startCaptureOrTriggerUpload(event: MouseEvent) {
     if (event.target === inputRef.current) {
-      if (forceNativeCamera) {
+      if (nativeCameraABTestingEnabled) {
+        trackEvent('IdV: Native camera A/B Test', {
+          native_camera_only: nativeCameraOnly,
+        });
+        if (nativeCameraOnly) {
+          return forceUpload();
```

Comment:
I suspect this may be prone to the same sort of issue discovered at https://github.com/18F/identity-idp/pull/6909#issuecomment-1246840680 , and I'd propose a solution similar to the one at https://github.com/18F/identity-idp/pull/6909#discussion_r972028497 .

Reply:
Thanks for the catch! Done in 00a7de865